### PR TITLE
    WSTEAM1-1348  Fix service logo responsiveness and margins

### PR DESCRIPTION
**Pull request by Louis-Matsika**
Resolves JIRA [WSTEAM1-1348](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1348)

Overall changes
======
Resize longer service logos at lower break points.
Fixed service logo padding above group 2

**Additional work:**
Fixed missaligned text in footer

Testing
======
1. vist service with longer service name (e.g. russian, http://localhost:7080/russian).
2. check to see if service logo scales corretly by lowering the breakpoint

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/legacy/containers/Brand/index.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const BrandContainer = ({
   const svgRatio = brandSVG && brandSVG.ratio;
   const minWidth = svgRatio * svgMinHeight;
   const maxWidth = svgRatio * svgMaxHeight;
+  const longBrands = ['russian', 'kyrgyz', 'afaanoromoo', 'azeri', 'serbian'];
```
Comment (karinathomasbbc) [inline src/app/legacy/containers/Brand/index.jsx]: **nitpick**: Could this list be in alphabetical order 🙏 ?

**Review by karinathomasbbc — APPROVED** [review]

**Comment by amoore108** [comment]
Just double checking the Chromatic snapshots as there is a change:
Screenshot 2024-10-29 at 16 00 45

The one on the right is this PR. Is that change expected? The padding around the brand looks better now in this PR, but its no longer aligned to the content in the footer.

**Comment by Louis-Matsika** [comment]
> Just double checking the Chromatic snapshots as there is a change: Screenshot 2024-10-29 at 16 00 45
> 
> The one on the right is this PR. Is that change expected? The padding around the brand looks better now in this PR, but its no longer aligned to the content in the footer.

hmm, good catch. I'll double check with Wendy

**Review by amoore108 — APPROVED** [review]
Nice one, thanks for this 👍

**Pull request merged**